keep/api/routes/workflows.py

```diff
@@ -1070,7 +1070,7 @@ def get_workflow_execution_status(
 ) -> WorkflowExecutionDTO:
     tenant_id = authenticated_entity.tenant_id
     workflowstore = WorkflowStore()
-    workflow_execution = workflowstore.get_workflow_execution(
+    workflow_execution, logs = workflowstore.get_workflow_execution_with_logs(
```

Comment:
would just do
```suggestion
    workflow_execution = workflowstore.get_workflow_execution(...)
    workflow_execution_logs = workflowstore.get_workflow_execution_logs(...)
```
imho it's more elegant, but nvm